src/react-components/room/PlacePopover.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ButtonGridPopover } from "../popover/ButtonGridPopover";
+import { Popover } from "../popover/Popover";
+import { ToolbarButton } from "../input/ToolbarButton";
+import { ReactComponent as ObjectIcon } from "../icons/Object.svg";
+
+export function PlacePopoverButton({ items, onSelect }) {
+  // The button is removed if you can't place anything.
+  if (items.length === 0) {
+    return undefined;
+  }
+
+  return (
+    <Popover
+      title="Place"
+      content={props => <ButtonGridPopover items={items} onSelect={onSelect} {...props} />}
+      placement="top"
+      offsetDistance={28}
+      initiallyVisible
+    >
+      {({ togglePopover, popoverVisible, triggerRef }) => (
+        <ToolbarButton
+          ref={triggerRef}
+          icon={<ObjectIcon />}
+          selected={popoverVisible}
+          onClick={togglePopover}
+          label="Place"
```

Comment:
Not sure what the best word for it would be ("create", "sparn", "share", "post" and "add" all have their own issues), but "place" is very surprising to me -- might be an issue worth revisiting